setup.py

```diff
@@ -388,7 +480,7 @@ def pre_run(self):
 
 ext_modules = [
     create_extension(ONNXCpp2PyExtension,
-                     str('onnx.onnx_cpp2py_export'),
+                     str('libonnx_cpp2py_export'),
```

Comment:
Can we have a name without `lib` just to indicate that it is a python module?

Reply:
maybe. building with cmake via `add_library(foo)` forces it to be called 'libfoo' as far as I can tell - do you know how to disable that?

Reply:
Try this? `set_target_properties(onnx_cpp2py_export PROPERTIES PREFIX "")`